projects/kit/components/input-date-multi/input-date-multi.directive.ts

```diff
@@ -46,12 +47,25 @@ import {tuiMaskito} from '@taiga-ui/kit/utils';
     },
 })
 export class TuiInputDateMultiDirective extends TuiInputChipBaseDirective<TuiDay> {
-    private readonly dateOptions = inject(TUI_INPUT_DATE_OPTIONS_NEW);
+    private readonly dateOptions = inject(TUI_INPUT_DATE_MULTI_OPTIONS);
 
     protected readonly icon = tuiTextfieldIcon(TUI_INPUT_DATE_OPTIONS_NEW);
     protected readonly filler = tuiWithDateFiller();
-    protected readonly stringify = this.handlers.stringify.set((item) =>
-        item.toString(this.format().mode, this.format().separator),
+    protected readonly stringify = this.handlers.stringify.set(
```

Comment:
This code should NOT be dependent on the type, component only works with `TuiDay`.